templates/default/django_settings.py.erb

```diff
@@ -41,142 +43,82 @@ LANGUAGE_CODE = 'en-us'
 
 SITE_ID = 1
 
+#Google analytics ID. Enter the full id, as in: UA-xxxxxx-x 
+GOOGLE_ID = 'UA-9880071-1'
```

Comment:
If you can put the full ID in the databag (the 'UA-...' string) that would be great!